### PR TITLE
Add Ubuntu Noble support notes

### PR DESCRIPTION
**Pull request by mvitale1989**
#### Summary

Add Ubuntu Noble support

**NB**: this can be merged only after the next release, either patch or minor, and we have verified that server and omnibus work fine on Ubuntu Noble.

#### Ticket Link

https://mattermost.atlassian.net/browse/CLD-7799

### CODE REVIEW

File: source/install/common-prod-deploy-omnibus.rst [hunk 1]
```diff
@@ -3,7 +3,7 @@
...
 
-Mattermost bundles the components of a Mattermost deployment into a single installation, called **Omnibus**. Mattermost Omnibus currently supports Ubuntu's ``focal`` (20.04) and ``jammy`` (22.04) distributions. The package bundles the free, unlicensed Mattermost Enterprise version of Mattermost, and leverages the `apt package manager <https://ubuntu.com/server/docs/package-management>`__ to install and update the platform components. A custom CLI and ansible recipes link the components together and configures them.
+Mattermost bundles the components of a Mattermost deployment into a single installation, called **Omnibus**. Mattermost Omnibus currently supports Ubuntu's ``focal`` (20.04), ``jammy`` (22.04) and ``noble`` (24.04) distributions. The package bundles the free, unlicensed Mattermost Enterprise version of Mattermost, and leverages the `apt package manager <https://ubuntu.com/server/docs/package-management>`__ to install and update the platform components. A custom CLI and ansible recipes link the components together and configures them.
```
Comment (cwarnermm) [inline source/install/common-prod-deploy-omnibus.rst:6]:
```suggestion
Mattermost bundles the components of a Mattermost deployment into a single installation, called **Omnibus**. Mattermost Omnibus currently supports Ubuntu's ``focal`` (20.04), ``jammy`` (22.04), and ``noble`` (24.04) distributions. The package bundles the free, unlicensed Mattermost Enterprise version of Mattermost, and leverages the `apt package manager <https://ubuntu.com/server/docs/package-management>`__ to install and update the platform components. A custom CLI and ansible recipes link the components together and configures them.
```

**Review by cwarnermm — APPROVED** [review]
Thank you, @mvitale1989!!

CC @nab-77 - Look how easy it is to add distribution support now!

**Review by saturninoabril — APPROVED** [review]

**Comment by cwarnermm** [comment]
Friendly reminder @toninis @phoinixgrr that this docs PR will benefit from your reviews. Thanks!

**Review by toninis — APPROVED** [review]

**Review by phoinixgrr — APPROVED** [review]
LGTM 🚀 
:shipit:

**Comment by cwarnermm** [comment]
@mvitale1989 - Is this being tested as part of v9.11 release readiness? Or should we target a different milestone to publish this docs PR update?

**Comment by mvitale1989** [comment]
@cwarnermm All test servers are upgraded to Ubuntu Noble, and i have run the automated E2E Cypress tests on Ubuntu Noble as well, it is on track for v9.11 :+1: 
cc @amyblais @saturninoabril FYI

**Comment by cwarnermm** [comment]
Fantastic! Thanks, @mvitale1989!!

**Comment by saturninoabril** [comment]
Thanks @mvitale1989, great work in adding support to Noble and ensuring it's fully tested!

**Pull request merged**